crypto/x509/x509_req.c

```diff
@@ -278,21 +278,21 @@ X509_NAME *X509_REQ_get_subject_name(const X509_REQ *req)
 }
 
 void X509_REQ_get0_signature(const X509_REQ *req, const ASN1_BIT_STRING **psig,
-                             const X509_ALGOR **palg)
+                             const X509_ALGOR **ppalg)
```

Comment:
Sorry, I shouldn't have mentioned the Hungarian notation. I was actually just pointing out in a side note that it was used inconsistently and therefore useless. I didn't expect expect that my remark  would inspire you to change `palg` to `ppalg`. Your change made things a little bit more inconsistent than they was before, because now the function prototype has a `**psig` with a single 'p' and a `**ppalg` with a double 'p'.

Also the inconsistency between the prototypes of the three similar functions below has increased. (The worst inconsistency being the different ordering of the arguments in `X509_get0_signature()`, sigh.).

https://github.com/openssl/openssl/blob/6eac9d95cf75964c8c80ed7bd15f2651a6b3b888/crypto/x509/x509_req.c#L280-L281

https://github.com/openssl/openssl/blob/6eac9d95cf75964c8c80ed7bd15f2651a6b3b888/crypto/x509/x509cset.c#L121-L122

https://github.com/openssl/openssl/blob/6eac9d95cf75964c8c80ed7bd15f2651a6b3b888/crypto/x509/x_x509.c#L243-L244


So I would ask you to revert the `ppalg` back to a `palg`, sorry for the inconvenience. The rest is fine, I'll approve as soon as the `ppalg` is reverted.

Sorry for the inconvenience I caused :-/

Reply:
No worries. Reverted & pushed.